Early-stage startups need analytics, here’s why (and what matters most) (#5237)
PostHog/posthog.com: contents/blog/early-stage-analytics.md

---

> title: "Early-stage startups need analytics, here’s why (and what matters most)"

Reviewer: I quite like 'The 80/20 of early-stage startup analytics' for the actual headline... thoughts? It's nice and punchy.

Author: I like it more too

date: 2023-02-07
author: ["ian-vanagas"]
showTitle: true
rootpage: /blog
sidebar: Blog
hideAnchor: true
featuredImage: ../images/blog/posthog-blog-image.png
featuredImageType: full
category: Startups
tags:
 - Guides
---

> Starting a startup is trying to find the exit of a dark cave. You attempt a lot of things, but it is mostly dark. Sometimes you hear sounds leading you somewhere, but they can also be misleading. In this cave, analytics is like a flashlight. It doesn’t automatically show you the exit, but it does provide clarity on where you are going. With the light of analytics, you stumble less and spend less time moving in the wrong direction.

Reviewer: ```suggestion
Building a startup is like trying to find the exit of a dark cave. You attempt a lot of things, but it's mostly dark. Sometimes you hear sounds leading you somewhere, but they can also be misleading. In this cave, analytics is like a flashlight. It doesn’t automatically show you the exit, but it helps you see where you're going. With the light of analytics, you stumble less and spend less time moving in the wrong direction.
```

![Flashlight](../images/blog/early-stage-analytics/light.png)

> Our team works with a lot of high-growth startups (they are [our ideal customers](/handbook/strategy/ideal-customer-persona)). We succeed when they succeed. We believe analytics is a key part of finding early success if done right. This post goes over why we think your startup needs analytics, and what exactly those analytics should look like at an early stage.

Reviewer: ```suggestion
Our team works with a lot of high-growth startups (they are [our ideal customers](/handbook/strategy/ideal-customer-persona)). We succeed when they succeed. We know from experience that understanding your users is a vital part of finding early success, and you can't do that without analytics. Here we explain and how to use analytics effectively at an early stage.
```

> ## Why do analytics matter for startups?

Reviewer: This section is ok, but it would be with some more concrete examples either from our own experience, or from those of our users.

Author: Added one on OKRs

> 
> Analytics provide insight into how your users are using your product. They are direct feedback to help you figure out what to work on. Like a flashlight, they are a light for the dark journey to product market fit.

Reviewer: ```suggestion
```

Next para feels a bit more to the point.

> As a simple example, product analytics show you whether your users are actually using your product. Not just "oh yeah, I’ll check it out," but seeing if they are logging in, using the features you developed, and most importantly, returning to use your product. Analytics show what areas they are finding value in, what needs to improve, and what is broken.

Reviewer: ```suggestion
As a simple example, product analytics show you whether your users are actually using your product. Not just "oh yeah, I’ll check it out," but seeing if they are logging in, using the features you developed, and most importantly, returning to use them again. Analytics show what areas they are finding value in, what needs to improve, and what is broken.
```

> From this usage, you can begin creating insights to help you track key metrics like traction, engagement, active users, revenue, and feature usage. Without this, you rely heavily on anecdotal evidence. You simply want to see these numbers go up and to the right, there’s not a lot of deep analysis needed here. If the metrics aren’t moving in the right direction, it is a sign to change.

Reviewer: ```suggestion
From this usage, you can begin creating insights to help you track [key product metrics](/blog/b2b-saas-product-metrics) like traction, engagement, active users, revenue, and feature usage. Without this, you rely heavily on anecdotal evidence. You simply want to see these numbers go up and to the right, there’s not a lot of deep analysis needed here. If the metrics aren’t moving in the right direction, it is a sign to change.
```

> Startups are defined by change. They are all about testing, getting feedback rapidly, and constantly improving, even if that means change. Real usage analytics is rapid feedback. As soon as an action is completed, you know about it because of analytics. Time matters a lot for early-stage startups, and analytics help you make the most out of it.

Reviewer: ```suggestion
And startups are defined by change. They're all about testing, rapid feedback, and constant improvement. Real usage analytics is the most rapid feedback there is, so starting early is a competitive advantage you shouldn't miss.
```

Feel free to drop the 'And' here, but I feel like it help connect the previous section.

## The 80/20 of early-stage startup analytics

> As much as we value analytics, we still think building an actual product is the most important. No amount of time spent analyzing metrics is going to make them grow faster than building a better product. Because time is so valuable for startups, here’s what we recommend startups do to get the most benefit from analytics in the shortest amount of time.

Reviewer: Can we say this shorter? Like one or two sentences max. Just feels like filler atm.

### 1. Set up and watch session recordings

> Session recordings are the analytics tool that provide the most benefits to early-stage startups. They help you learn exactly how early users are using your product on an individual basis. This uncovers the usage patterns in key areas of the product as well as big problems. For example, users often use the product in unexpected or incorrect ways, leading to unexpected insights.

Reviewer: ```suggestion
Session recordings are the most useful analytical tool for early-stage startups. They help you learn exactly how early users are using your product, uncovering usage patterns, obvious failure modes, and unexpected behaviors. big problems.
```

> PostHog makes it easy to set up session recordings, just include the [snippet or library](/docs/integrate) in your app and turn them on in project settings. Once done, you get a steady stream of session recordings (if you have users) in your PostHog instance. 

Reviewer: ```suggestion
PostHog includes 15k monthly recordings for free – more than enough for any early-stage product. Just include the [snippet or library](/docs/integrate) in your app and turn them on in project settings. Once done, you get a steady stream of session recordings (if you have users) in your PostHog instance. 
```

> To maximize your problem-solving potential, you can also add [Sentry](/docs/integrate/third-party/sentry) and connect it to PostHog to monitor errors. When Sentry captures an error, the PostHog connection links the session recording. The recording becomes the re-creation steps that make squashing bugs and fixing issues easier.

Reviewer: ```suggestion
To maximize your problem-solving potential, add [Sentry](/docs/integrate/third-party/sentry) and connect it to PostHog to monitor errors. When Sentry captures an error, the PostHog connection links the session recording. The recording becomes the re-creation steps that make squashing bugs and fixing issues easier.
```

That last sentence is super hard to parse, can you have a go at re-wording?

Session recordings replace number-crunching, database querying, and user testing to give you the details on the good and bad of your product fast.

### 2. Build a key metrics dashboard

> As mentioned, analytics enable startups to understand if users are using the product. To keep this top of mind, you can build a key metrics dashboard. This keeps metrics like signups, pageviews, key feature usage, and revenue all in one place for the team.

Reviewer: ```suggestion
A key metrics dashboard is useful for validating users are actually using the product. This keeps metrics like signups, pageviews, key feature usage, and revenue all in one place for the team.
```

Maybe remove revenue? Might too early for that.

### 4. Monitor key actions and trigger processes

Once analytics, dashboards, and a process for getting feedback from users are in place, you can begin to leverage analytics in your business processes. This means triggering a process like reaching out to potential customers on signup or helping high-value users when they have trouble with a specific feature.

The analytics become a part of your business processes. PostHog helps this happen by enabling an action to trigger a webhook. This webhook can then connect to services like Zapier, [Slack](/docs/integrate/webhooks/slack), and [Discord](/tutorials/how-to-connect-discord-to-posthog-with-zapier) to trigger your team to reach out, update settings, offer support, or other processes.

## The balance of early-stage startup analytics

> Analytics at early-stage startups is a balance. You want an understanding of your product, without being bogged down by overanalysis. Startups need enough information to know what to build next, and plenty of time to build it. The analytics tools outlined here are what we found to work for early-stage startups we work with, but every case is different.

Reviewer: ```suggestion
Analytics at early-stage startups is a balance. You need just enough information to know what to build next, without getting bogged down by overanalysis. The approached outlined here are what we found to work for early-stage startups we work with, but every case is different.
```

> Analytics at early-stage startups is a balance. You want an understanding of your product, without being bogged down by overanalysis. Startups need enough information to know what to build next, and plenty of time to build it. The analytics tools outlined here are what we found to work for early-stage startups we work with, but every case is different.
> 
> Session recordings, key metric dashboards, feedback gathering, and key action monitoring strike the balance. They give early-stage startups the biggest benefit for the lowest investment. These analytics tools provide the light startups need to work their way out of the early-stage cave to product market fit. If this sounds helpful, [sign up for PostHog for free](https://app.posthog.com/signup).

Reviewer: ```suggestion
Analytics at early-stage startups is a balance. You need just enough information to know what to build next, without getting bogged down in overanalysis. Follow these steps and you'll have a built an efficient flashlight to find your way to product-market fit. If this sounds helpful, [sign up for PostHog for free](https://app.posthog.com/signup).
```

---
title: "The 80/20 of early-stage startup analytics"

> date: 2023-02-07

Reviewer: ```suggestion
date: 2023-02-09
```

And startups are defined by change. They're all about testing, rapid feedback, and constant improvement. Real usage analytics is the most rapid feedback there is, so starting early is a competitive advantage you shouldn't miss.

For example, we at PostHog use analytics to track OKRs, debug issues, test new features, discover opportunities, and more. To narrow in on OKRs, every team connects them to analytics:

- Team session recordings have "percentage of users per organization viewing recordings"
- Team product analytics have "top 10 US & EU clients 95th percentile of dashboard load time is under 5 seconds"
- Team experimentation have "double the number of organizations that use feature flags"

These metrics provide clear, rapid feedback on whether what we are building is valuable and making a difference. Without this, we would spend more time directionless and building features that might not matter to our users.

## The 80/20 of early-stage startup analytics

> As much as we value analytics, we believe the best way to create a successful startup is to spend time building a better product, not analytics analysis. To ensure you have time to build, here's what we recommend to get the most benefit from analytics in the shortest time.

Reviewer: "analytics analysis" is brain melter. Need an alternative.. "not obsessing over data"?

> As much as we value analytics, we believe the best way to create a successful startup is to spend time building a better product, not analytics analysis. To ensure you have time to build, here's what we recommend to get the most benefit from analytics in the shortest time.

Author: ```suggestion
As much as we value analytics, we believe the best way to create a successful startup is to spend time building a better product, not obsessing over data. To ensure you have time to build, here's what we recommend to get the most benefit from analytics in the shortest time.
```
